### DISCUSSION TITLE
Your Results in New Super-Resolution Benchmarks

### DISCUSSION THREAD

**Discussion by EvgeneyBogatyrev** [open-mmlab/mmediting#1223] [General]
Hello,

[MSU Graphics & Media Lab Video Group](https://videoprocessing.ai/) has recently launched two new Super-Resolution Benchmarks.
* [Video Upscalers Benchmark: Quality Enhancement](https://videoprocessing.ai/benchmarks/video-upscalers.html) determines the best upscaling methods for increasing video resolution and improving visual quality.
* [Super-Resolution for Video Compression benchmark](https://videoprocessing.ai/benchmarks/super-resolution-for-video-compression.html) aims to test Super-Resolution methods on compressed videos and select the best model for each video codec standard.

Your method BasicVSR++ achieved 22nd place in [Video Upscalers Benchmark: Quality Enhancement](https://videoprocessing.ai/benchmarks/video-upscalers.html) in 'Animation 2x' category. We look forward to your future work!

We would be grateful for your feedback on our work.

**Comment by AGenchev** [comment]
@EvgeneyZ How long sequence you fed into the model ? 
Did you fed one sequence after another or you implemented kind of a sliding window (with overlapping) approach ?

**Comment by EvgeneyBogatyrev** [comment]
We used a sliding window with overlapping. If you get better results by applying your model to our test video differently, please send us this result, and we will update your position in the leaderboard.
